spec/controllers/api/v1/cookie_and_jwt_api/protocol_subscriptions_controller_spec.rb

```diff
@@ -61,4 +61,59 @@
       end
     end
   end
+
+  describe 'create' do
+    describe 'without cookie' do
+      it 'returns 401 if no protocol_subscription is available' do
```

Comment:
why should this be a 401 btw? Isnt that something like unauthorized? Wouldnt 400 be better?

Reply:
Yes, sorry the spec was named wrong.